src/password/Create.vue

```diff
@@ -7,32 +7,53 @@
 
       <v-form @submit.prevent="save" ref="form">
         <p>
-          {{ $vuetify.lang.t('$vuetify.password.create.username', $root.idpConfig.idpName) }} <strong class="body-2">{{ $user.idp_username }}</strong>
+          {{ $vuetify.lang.t('$vuetify.password.create.username', $root.idpConfig.idpName) }}
+          <strong class="body-2">{{ $user.idp_username }}</strong>
         </p>
 
         <div class="password">
-          <BaseTextField :type="passwordIsHidden ? 'password' : 'text'" :label="$vuetify.lang.t('$vuetify.password.create.pwInput')" v-model="password" 
-          :rules="rules" :error-messages="errors" validate-on-blur @keyup.enter="blur" autofocus name="password" />
-          
+          <BaseTextField
+            :type="passwordIsHidden ? 'password' : 'text'"
+            :label="$vuetify.lang.t('$vuetify.password.create.pwInput')"
+            v-model="password"
+            :rules="rules"
+            :error-messages="errors"
+            validate-on-input
```

Comment:
this line is the only non-formatting change

Reply:
Thanks for calling out this line. It would have been a needle in a haystack! Would you add a YT issue to reformat the entire repo? It would be nice to not have this issue in the future.
How does `validate-on-input` work? Is it a Vue feature?

Reply:
Already on it. Yes, `validate-on-input` is the vue equivalent to `on:input={validate}` in Svelte, except its really part of the [Vuetify component BaseTextField](https://vuetifyjs.com/en/api/v-text-field/#props-validate-on).

Reply:
To be more precise its a prop that tells the component to validate internally